reference/SimpleCalculator/SimpleCalculator.Tests/AppInfoTests.cs

```diff
@@ -0,0 +1,112 @@
+using SimpleCalculator.Business;
+
+namespace SimpleCalculator.Tests;
+
+public class AppInfoTests
+{
+	[SetUp]
+	public void Setup()
+	{
+	}
+
+    [Test]
```

Comment:
Please use parameterized tests for this (reach out to me if you need a hand with creating them)